Package.swift

```diff
@@ -14,7 +19,7 @@ let package = Package(
     ],
     dependencies: [
         // Dependencies declare other packages that this package depends on.
-        .package(url: "https://github.com/CodeSlicing/pure-swift-ui.git", .exact("2.0.1")),
+        .package(url: "https://github.com/Brett-Best/pure-swift-ui.git", .branch("feature/MPP-Support")),
```

Comment:
This will need changing once https://github.com/CodeSlicing/pure-swift-ui/pull/3 is merged.

Reply:
Before merging can you change this back to CodeSlicing and .branch("feature/3.0.0") please.